packages/chain-adapters/src/utxo/types.ts

```diff
@@ -74,7 +74,7 @@ export type FeeData = {
 }
 
 export type BuildTxInput = {
-  from?: string
+  from?: string // used for savers, the address to consume the utxos
```

Comment:
```suggestion
  from?: string
```

Reply:
Actually let's keep this comment, turbo confusing for someone else reading this